### DISCUSSION TITLE
Achievement: Pull requests count is incorrect

### DISCUSSION THREAD

**Discussion by ctjlewis** [community/community#18703] [Other Feature Feedback, Questions, & Ideas] [answered]
Just noticed these on my profile - definitely have had more than 3 PRs merged though.

**Comment by thomscoder** [comment, marked as answer]
Hi @ctjlewis 
That x3 is most likely your tier.
- Tier 1: Base
- Tier 2: Bronze
- Tier 3: Silver
- Tier 4: Gold

According to your README.md you have opened something like 100+ PR that got you to unlock the Silver rank - Tier 3.
For the gold rank (Tier 4) you need around 1024+ PRs

You can consult the tiering list here: https://github.com/Schweinepriester/github-profile-achievements

**Reply by ctjlewis** [reply]
Thanks, that makes a lot more sense! “x3” text probably not the best choice from a UX standpoint.

**Reply by aminya** [reply]
This is currently confusing. I think using the PR count is a better measure than some custom tier.

**Comment by cyai** [comment]
The answer is in this link https://github.com/orgs/github-community/discussions/18156 😄

Actually, I too had the same problem, after some research I came to the following discussion and got my answer clear!
